server/block/slime.go

```diff
@@ -0,0 +1,21 @@
+package block
+
+// A Slime block is a translucent block that entities can bounce on
```

Comment:
Just to stay consistent, I'd remove the `A ` prefix and add a period at the end of the sentence.

Reply:
I do that

Reply:
You might as well also remove the `block` suffix after `Slime`.

Reply:
done